factory/bootstrap/bootstrapComponents.go

```diff
@@ -168,6 +168,11 @@ func (bcf *bootstrapComponentsFactory) Create() (*bootstrapComponents, error) {
 
 	dataSyncerFactory := bootstrap.NewScheduledDataSyncerFactory()
 
+	// increment num received to make sure that first heartbeat message
+	// will have value 1, thus explorer will display status in progress
+	tss := bcf.statusCoreComponents.TrieSyncStatistics()
+	tss.AddNumReceived(1)
```

Comment:
I think we should use processed instead of received.

Reply:
wrong naming

Reply:
as discussed, will change the naming of the methods in a separate PR

Reply:
updated into #4614